nextflow_schema.json

```diff
@@ -400,6 +400,22 @@
             "default": "",
             "fa_icon": "fas fa-copy",
             "properties": {
+                "cadd_prescored": {
+                    "type": "string",
+                    "exists": true,
+                    "format": "directory-path",
+                    "fa_icon": "fas fa-file",
+                    "description": "Path to the directory containing cadd prescored files.",
+                    "help_text": "This folder contains the compressed files and indexes that would otherwise be in data/prescored folder as described in https://github.com/kircherlab/CADD-scripts/#manual-installation."
```

Comment:
Should we indicate that, as written currently, this file is only used to score the known indels. Might be worth clarifying that this is not the prescored snv file.